ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -50,8 +50,16 @@ def _es_modules():
             pass
 
 
-def get_version(elasticsearch):
-    return getattr(elasticsearch, "__version__", (0, 0, 0))
+versions = {}
+
+def get_version_tuple(elasticsearch):
+    if getattr(elasticsearch, "__name__", None):
+        versions[elasticsearch.__name__] = getattr(elasticsearch, "__versionstr__", "0.0.0")
+    return getattr(elasticsearch, "__version__", "0.0.0")
+
+
+def get_version():
+    return versions
```

Comment:
I think the fact that this is a dictionary will cause problems for `add_integration`, which expects a string argument.

Reply:
I updated the monkey patch code to account for this when calling `module.get_version()`. I think it's important that we include the exact package that was patched, in this case elasticsearch2, elasticsearch3 .... and it's corresponding version.